src/ParallelAlgorithms/SurfaceFinder/SurfaceFinder.cpp

```diff
@@ -29,8 +29,8 @@ struct RayInterpolant {
     return apply_matrices(interpolation_matrices, values, mesh.extents())[0];
   }
 
-  const DataVector& values{};
-  const Mesh<1>& mesh{};
+  const DataVector& values;
+  const Mesh<1>& mesh;
```

Comment:
[optional] Either
1. NOLINT these
2. Make the mesh not a ref (it's computed using `slice_through` anyways) and the DataVector non-owning
3. Use pointers

Reply:
@nilsvu I think this is code for ID or something? I don't know what it's supposed to do so could you take over addressing this? I remember looking at the code and the current use-case is technically fine.

Reply:
Good enough for me